dom.bs

```diff
@@ -358,34 +358,24 @@ use of this functionality through <a>events</a>
 (synthetic events) can make use of the return value of the
 {{EventTarget/dispatchEvent()}} method:
 
-<pre class='lang-javascript'>
+<pre class=lang-javascript>
 if(obj.dispatchEvent(event)) {
   // event was not canceled, time for some magic
   &hellip;
 }
 </pre>
 
-When an <a>event</a> is
-<a>dispatched</a> to an object that
-<a>participates</a> in a
-<a>tree</a> (e.g. an
-<a for="/">element</a>), it can reach
-<a>event listeners</a> on that object's
-<a>ancestors</a> too. First all object's
-<a>ancestor</a>
-<a>event listeners</a> whose
-<a for="event listener">capture</a> variable is set to true are invoked, in
-<a>tree order</a>. Second, object's own
-<a>event listeners</a> are invoked. And
-finally, and only if <a>event</a>'s
-{{Event/bubbles}} attribute value is true,
-object's  <a>ancestor</a>
-<a>event listeners</a> are invoked again,
-but now in reverse <a>tree order</a>.
-
-Let's look at an example of how <a>events</a> work in a <a>tree</a>:
-
-<pre class='lang-markup'>
+<p>When an <a>event</a> is <a>dispatched</a> to an object that <a>participates</a> in a <a>tree</a>
+(e.g., an <a for=/>element</a>), it can reach <a>event listeners</a> on that object's
+<a>ancestors</a> too. Effectively, all the object's <a>inclusive ancestor</a> <a>event listeners</a>
```

Comment:
The use of the word "Effectively" leaves ambiguity as to what the true behavior is. If there is in fact no ambiguity, then I would not use that word; I would replace it with "Firstly" or "In particular" or something like that. If there is in fact ambiguity (I do not know) then it's fine to leave it, though a note of clarification may be helpful.

Reply:
It's an approximation as shadow trees and the `Window` object make it more involved than this. Not sure how to capture that succinctly though.

Reply:
Ok, that's fine then.